news/_posts/core-weekly/2021-06-07-coreweekly-22-2021.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 22 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-06-07
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 31th of May to Sunday 6th of June 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## A quick update about PrestaShop's GitHub issues and pull requests:
+
+- [63 new issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+created%3A2021-05-31..2021-06-06) have been created in the project repositories;
+- [40 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2021-05-31..2021-06-06), including [20 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2021-05-31..2021-06-06) on the core;
+- [99 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2021-05-31..2021-06-06) in the project repositories;
+- [87 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2021-05-31..2021-06-06), including [66 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2021-05-31..2021-06-06).
+
+
+
+## Code changes in the 'develop' branch
+
+
+### Core
+* [#24782](https://github.com/PrestaShop/PrestaShop/pull/24782): Check that $session is set before using it. Thank you [@tswfi](https://github.com/tswfi)
+* [#24760](https://github.com/PrestaShop/PrestaShop/pull/24760): Drop leftover code, /tools/http_build_url/ was removed. Thank you [@mvorisek](https://github.com/mvorisek)
+* [#24757](https://github.com/PrestaShop/PrestaShop/pull/24757): Make getValue more strict to avoid false negative. Thank you [@RomainMazB](https://github.com/RomainMazB)
```

Comment:
```suggestion
* [#24757](https://github.com/PrestaShop/PrestaShop/pull/24757): Make `getValue` more strict to avoid false negative. Thank you [@RomainMazB](https://github.com/RomainMazB)
```